### ISSUE TITLE
can't activate in git-bash

### ISSUE THREAD

**Issue by hsharrison** [conda/conda#6544]
I'm a Linux user so Windows is new to me. I'm running git-bash in ConEmu.

```
source activate python2
bash: /c/ci/conda_1513927372348/_h_env/etc/profile.d/conda.sh: No such file or directory
```

No idea where that path is coming from. conda and activate are in AppData.

**Comment by howach** [comment]
I'm having the same issue. Happened after I upgraded Anaconda from 3.5 to 3.6 with "conda update python"

**Comment by kalefranz** [comment]
What is the output of

    conda info
    which -a activate
    cat $(which activate)

**Comment by howach** [comment]
I'm using Gir for Windows with Bash emulation in cmd.

 `conda info`
    active environment : None
       user config file : C:\Users\waho\.condarc
 populated config files :
          conda version : 4.4.3
    conda-build version : 3.0.27
         python version : 3.6.3.final.0
       base environment : C:\Users\waho\AppData\Local\Continuum\anaconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/main/win-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
          package cache : C:\Users\waho\AppData\Local\Continuum\anaconda3\pkgs
                          C:\Users\waho\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\waho\AppData\Local\Continuum\anaconda3\envs
                          C:\Users\waho\AppData\Local\conda\conda\envs
                          C:\Users\waho\.conda\envs
               platform : win-64
             user-agent : conda/4.4.3 requests/2.18.4 CPython/3.6.3 Windows/10 Windows/10.0.15063
          administrator : False
             netrc file : None
           offline mode : False

`which -a activate`
/c/Users/waho/AppData/Local/Continuum/anaconda3/Scripts/activate

`cat $(which activate)`
cat: '$(which': No such file or directory
cat: 'activate)': No such file or directory

**Comment by hsharrison** [comment]
Sorry for the delay, had to wait until I was back at work.

````
HAR9BQ@P17-0820 MINGW64 ~
$ conda info

     active environment : None
            shell level : 0
       user config file : C:\Users\HAR9BQ\.condarc
 populated config files : C:\Users\HAR9BQ\.condarc
          conda version : 4.4.2
    conda-build version : 3.2.0
         python version : 3.6.3.final.0
       base environment : C:\Users\HAR9BQ\AppData\Local\Continuum\anaconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/main/win-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
          package cache : C:\Users\HAR9BQ\AppData\Local\Continuum\anaconda3\pkgs
                          C:\Users\HAR9BQ\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\HAR9BQ\AppData\Local\Continuum\anaconda3\envs
                          C:\Users\HAR9BQ\AppData\Local\conda\conda\envs
                          C:\Users\HAR9BQ\.conda\envs
               platform : win-64
             user-agent : conda/4.4.2 requests/2.18.4 CPython/3.6.3 Windows/7 Windows/6.1.7601
          administrator : True
             netrc file : None
           offline mode : False


HAR9BQ@P17-0820 MINGW64 ~
$ which -a activate
/c/Users/HAR9BQ/AppData/Local/Continuum/anaconda3/Scripts/activate
/c/Users/HAR9BQ/AppData/Local/Continuum/anaconda3/Scripts/activate

HAR9BQ@P17-0820 MINGW64 ~
$ cat $(which activate)
#!/bin/sh
_CONDA_ROOT="/c/ci/conda_1513927372348/_h_env"
#!/bin/sh

_conda_set_vars() {
    # set _CONDA_SHELL_FLAVOR
    if [ -n "${BASH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=bash
    elif [ -n "${ZSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=zsh
    elif [ -n "${KSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=ksh
    elif [ -n "${POSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=posh
    else
        # https://unix.stackexchange.com/a/120138/92065
        local _q="$(ps -p$$ -o cmd="",comm="",fname="" 2>/dev/null | sed 's/^-//' | grep -oE '\w+' | head -n1)"
        if [ "$_q" = dash ]; then
            _CONDA_SHELL_FLAVOR=dash
        else
            (>&2 echo "Unrecognized shell.")
            return 1
        fi
    fi

    # https://unix.stackexchange.com/questions/4650/determining-path-to-sourced-shell-script/
    local script_dir
    case "$_CONDA_SHELL_FLAVOR" in
        bash) script_dir="$(dirname "${BASH_SOURCE[0]}")";;
        zsh) script_dir="$(dirname "${(%):-%x}")";;  # http://stackoverflow.com/a/28336473/2127762
        dash) x=$(lsof -p $$ -Fn0 | tail -1); script_dir="$(dirname "${x#n}")";;
        *) script_dir="$(cd "$(dirname "$_")" && echo "$PWD")";;
    esac

    if [ -z "${_CONDA_ROOT+x}" ]; then
        _CONDA_ROOT="$(dirname "$script_dir")"
    fi

}

_conda_script_is_sourced() {
    # http://stackoverflow.com/a/28776166/2127762
    sourced=0
    if [ -n "${ZSH_EVAL_CONTEXT:+x}" ]; then
      case $ZSH_EVAL_CONTEXT in *:file) sourced=1;; esac
    elif [ -n "${KSH_VERSION:+x}" ]; then
      [ "$(cd $(dirname -- $0) && pwd -P)/$(basename -- $0)" != "$(cd $(dirname -- ${.sh.file}) && pwd -P)/$(basename -- ${.sh.file})" ] && sourced=1
    elif [ -n "${BASH_VERSION:+x}" ]; then
      [ "${BASH_SOURCE[0]}" = "$0" ] && sourced=1
    else # All other shells: examine $0 for known shell binary filenames
      # Detects `sh` and `dash`; add additional shell filenames as needed.
      case ${0##*/} in sh|dash) sourced=0;; *) sourced=1;; esac
    fi
    return $sourced
}

if ! _conda_script_is_sourced; then
    (
      >&2 echo "Error: activate must be sourced. Run 'source activate envname'"
      >&2 echo "instead of 'activate envname'."
    )
    exit 1
fi

_conda_set_vars
. "$_CONDA_ROOT/etc/profile.d/conda.sh" || return $?
_conda_activate "$@"
````

**Comment by kalefranz** [comment]
This should be fixed in recent versions of conda 4.4 (i.e. 4.4.7).  If not, please let me know and I'll re-open the issue.

**Comment by gustavovalverde** [comment]
Hi @kalefranz same issue using Git Bash on Windows 10:
### conda info
```
     active environment : None
       user config file : C:\Users\gvalv\.condarc
 populated config files : C:\Users\gvalv\.condarc
          conda version : 4.4.10
    conda-build version : 3.4.1
         python version : 3.6.4.final.0
       base environment : C:\ProgramData\Anaconda3  (read only)
           channel URLs : https://repo.continuum.io/pkgs/main/win-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
          package cache : C:\ProgramData\Anaconda3\pkgs
                          C:\Users\gvalv\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\gvalv\AppData\Local\conda\conda\envs
                          C:\ProgramData\Anaconda3\envs
                          C:\Users\gvalv\.conda\envs
               platform : win-64
             user-agent : conda/4.4.10 requests/2.18.4 CPython/3.6.4 Windows/10 Windows/10.0.16299
          administrator : False
             netrc file : None
           offline mode : False

```

### which -a activate
`/c/ProgramData/Anaconda3/Scripts/activate`

### cat $(which activate)

```shell
#!/bin/sh
_CONDA_ROOT="$(cygpath 'C:/ProgramData/Anaconda3')"
#!/bin/sh

_conda_set_vars() {
    # set _CONDA_SHELL_FLAVOR
    if [ -n "${BASH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=bash
    elif [ -n "${ZSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=zsh
    elif [ -n "${KSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=ksh
    elif [ -n "${POSH_VERSION:+x}" ]; then
        _CONDA_SHELL_FLAVOR=posh
    else
        # https://unix.stackexchange.com/a/120138/92065
        local _q="$(ps -p$$ -o cmd="",comm="",fname="" 2>/dev/null | sed 's/^-//' | grep -oE '\w+' | head -n1)"
        if [ "$_q" = dash ]; then
            _CONDA_SHELL_FLAVOR=dash
        else
            (>&2 echo "Unrecognized shell.")
            return 1
        fi
    fi

    # https://unix.stackexchange.com/questions/4650/determining-path-to-sourced-shell-script/
    local script_dir
    case "$_CONDA_SHELL_FLAVOR" in
        bash) script_dir="$(dirname "${BASH_SOURCE[0]}")";;
        zsh) script_dir="$(dirname "${(%):-%x}")";;  # http://stackoverflow.com/a/28336473/2127762
        dash) x=$(lsof -p $$ -Fn0 | tail -1); script_dir="$(dirname "${x#n}")";;
        *) script_dir="$(cd "$(dirname "$_")" && echo "$PWD")";;
    esac

    if [ -z "${_CONDA_ROOT+x}" ]; then
        _CONDA_ROOT="$(dirname "$script_dir")"
    fi

}

_conda_script_is_sourced() {
    # http://stackoverflow.com/a/28776166/2127762
    local sourced=0
    if [ -n "${ZSH_EVAL_CONTEXT:+x}" ]; then
      case $ZSH_EVAL_CONTEXT in *:file) sourced=1;; esac
    elif [ -n "${KSH_VERSION:+x}" ]; then
      [ "$(cd $(dirname -- $0) && pwd -P)/$(basename -- $0)" != "$(cd $(dirname -- ${.sh.file}) && pwd -P)/$(basename -- ${.sh.file})" ] && sourced=1
    elif [ -n "${BASH_VERSION:+x}" ]; then
      [ "${BASH_SOURCE[0]}" = "$0" ] && sourced=1
    else # All other shells: examine $0 for known shell binary filenames
      # Detects `sh` and `dash`; add additional shell filenames as needed.
      case ${0##*/} in sh|dash) sourced=0;; *) sourced=1;; esac
    fi
    return $sourced
}

if ! _conda_script_is_sourced; then
    (
      >&2 echo "Error: activate must be sourced. Run 'source activate envname'"
      >&2 echo "instead of 'activate envname'."
    )
    exit 1
fi

_conda_set_vars
. "$_CONDA_ROOT/etc/profile.d/conda.sh" || return $?
_conda_activate "$@"

```
